prov/efa/test/rdma_core_mocks.c

```diff
@@ -0,0 +1,22 @@
+
+#include <infiniband/efadv.h>
+#include <infiniband/verbs.h>
+#include <stdlib.h>
+#include <stdint.h>
+#include <stdarg.h>
+#include <stddef.h>
+#include <setjmp.h>
+#include <cmocka.h>
+
+struct ibv_ah *__wrap_ibv_create_ah(struct ibv_pd *pd, struct ibv_ah_attr *attr) {
+    check_expected(pd);
+    check_expected(attr);
+    return (struct ibv_ah*) mock();
```

Comment:
For clarification, can we add a comment with some details on what mock() is doing here?

Reply:
Done

Reply:
Thank-you